src/runtime/execution.rs

```diff
@@ -568,6 +570,35 @@ impl ExecutionState {
             self.current_span_entered = Some(unsafe { extend_span_entered_lt(self.current_span.enter()) });
         }
     }
+
+    fn set_tag_internal(&mut self, tag: u64) {
```

Comment:
This name is too vague.  Something like `set_tag_for_current_task` would be better.